models/classes/runner/QtiRunnerServiceContext.php

```diff
@@ -38,26 +47,37 @@
 use oat\taoQtiTest\models\cat\CatService;
 use oat\taoQtiTest\models\ExtendedStateService;
 use oat\taoQtiTest\models\SectionPauseService;
-use oat\tao\helpers\UserHelper;
+use oat\taoQtiTest\models\event\SelectAdaptiveNextItemEvent;
 use qtism\data\AssessmentTest;
 use qtism\data\AssessmentItemRef;
+use qtism\data\ExtendedAssessmentItemRef;
 use qtism\data\NavigationMode;
 use qtism\runtime\storage\binary\AbstractQtiBinaryStorage;
 use qtism\runtime\storage\binary\BinaryAssessmentTestSeeker;
 use qtism\runtime\tests\AssessmentTestSession;
+use qtism\runtime\tests\AssessmentTestSessionException;
 use qtism\runtime\tests\RouteItem;
-use oat\oatbox\event\EventManager;
-use oat\taoQtiTest\models\event\SelectAdaptiveNextItemEvent;
-use oat\libCat\result\ResultVariable;
+use tao_models_classes_service_FileStorage;
+use tao_models_classes_service_StorageDirectory;
+use taoQtiTest_helpers_SessionManager;
 use taoQtiTest_helpers_TestCompilerUtils;
+use taoQtiTest_helpers_TestRunnerUtils;
 use taoQtiTest_models_classes_QtiTestService;
+use taoResultServer_models_classes_Variable;
+use common_Exception;
+use common_exception_Error;
+use common_exception_NotImplemented;
+use common_Logger;
+use Exception;
 
 /**
  * Class QtiRunnerServiceContext
  *
  * Defines a container to store and to share runner service context of the QTI implementation
  *
  * @package oat\taoQtiTest\models
+ *
+ * @author Jean-Sébastien Conan <jean-sebastien.conan@vesperiagroup.com>
```

Comment:
?

Reply:
This was in a comment between the license and the imports, I've moved it as a class annotation.